posthog/models/personal_api_key.py

```diff
@@ -6,22 +9,30 @@
 
 # Fixed iteration count for PBKDF2PasswordHasher hasher.
 # This is the iteration count used by PostHog since the beginning of time.
-# Changing this would break all existing personal API keys.
+# Changing this would break all existing personal API keys using it still.
 PERSONAL_API_KEY_ITERATIONS = 260000
 
 PERSONAL_API_KEY_ITERATIONS_TO_TRY = (
-    PERSONAL_API_KEY_ITERATIONS,
-    390000,  # This is the iteration count used briefly on some API keys.
+    ("hash", None),  # Moved to simple hashing in 2024-02
+    ("pbkdf2", PERSONAL_API_KEY_ITERATIONS),
+    ("pbkdf2", 390000),  # This is the iteration count used briefly on some API keys.
 )
 
 # A constant salt is not nearly as good as user-specific, but we must be able to look up a personal API key
 # by itself. Some salt is slightly better than none though.
 PERSONAL_API_KEY_SALT = "posthog_personal_api_key"
 
 
-def hash_key_value(value: str, iterations: int = PERSONAL_API_KEY_ITERATIONS) -> str:
-    hasher = PBKDF2PasswordHasher()
-    return hasher.encode(value, PERSONAL_API_KEY_SALT, iterations=iterations)
+def hash_key_value(value: str, mode: str = "hash", iterations: Optional[int] = None) -> str:
+    if mode == "pbkdf2":
+        iterations = iterations or PERSONAL_API_KEY_ITERATIONS
```

Comment:
Feels like `PERSONAL_API_KEY_ITERATIONS` could be removed now I would have thought? It's only used by the `PERSONAL_API_KEY_ITERATIONS_TO_TRY` and probably a good thing to not have it there as a default.

Reply:
True, we don't need it as default anymore, it just needs to exist in the list of iterations to try.